src/universal/modules/teamDashboard/components/TeamCallsToAction/TeamCallsToAction.js

```diff
@@ -22,49 +24,61 @@ const ButtonGroup = styled('div')({
   display: 'flex',
   minWidth: '14rem',
   paddingLeft: ui.dashGutterSmall,
+
   [ui.dashBreakpoint]: {
     minWidth: '13rem',
     paddingLeft: ui.dashGutterLarge
   }
 });
 
-const ButtonBlock = styled('div')({
-  width: '100%'
-});
+const originAnchor = {
+  vertical: 'bottom',
+  horizontal: 'right'
+};
 
-const TeamCallToAction = ({history, teamId}: Props) => {
-  const goToMeetingLobby = () =>
-    history.push(`/meeting/${teamId}/`);
+const targetAnchor = {
+  vertical: 'top',
+  horizontal: 'right'
+};
 
-  const goToRetroLobby = () =>
-    history.push(`/retro/${teamId}/`);
+const TeamCallToAction = ({history, teamId}: Props) => {
+  const goToMeetingLobby = () => {
+    const slug = meetingTypeToSlug[ACTION];
+    history.push(`/${slug}/${teamId}/`);
+  };
+  const buttonToggle = (
+    <Button
+      buttonSize="small"
+      buttonStyle="primary"
+      colorPalette="warm"
+      icon="chevron-down"
+      iconPlacement="right"
+      isBlock
+      label="Start Meeting"
+    />
+  );
 
   return (
     <ButtonGroup>
-      <ButtonBlock>
+      {__RELEASE_FLAGS__.retro ?
+        <LoadableMenu
+          LoadableComponent={LoadableTeamCallsToActionMenu}
```

Comment:
The loadable menu wraps everything in a modal, determines where to place the menu on the screen, & animates it into view.